zds/utils/templatetags/joinby.py

```diff
@@ -0,0 +1,27 @@
+# coding: utf-8
+
+from django import template
+from django.utils.translation import ugettext_lazy as _
+
+register = template.Library()
+
+
+@register.simple_tag
+def joinby(values, separator=", ", same_final_separator=False):
+    """
+    Returns a human readable list (of type string) of values separated
+    by commas (excepted the last preceded by "et") from an iterable list.
+    Set 'same_final_separator' to True to make the last also preceded
+    by a comma.
+    """
+    if not values:
+        return ""
+    if len(values) == 1:
+        return values[0].__str__()
+    if same_final_separator:
+        final_sep = separator
+    else:
+        final_sep = unicode(_(" et "))
```

Comment:
Pour le `unicode`, c'est parce que j'avais une erreur étrange à la ligne d'après quand je concatenais `final_sep` avec l'autre string.

Voilà ce que j'obtenais : `TypeError: coercing to Unicode: need string or buffer, __proxy__ found`. Je n'ai pu résoudre ça qu'en rajoutant un `unicode()`, j'ignores pourquoi.